usagov/usagov-2021
# usagov-1839-cypress-nav-card-test: update test to run in English and …

…Spanish


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1839

## Description

Update the test run in English and Spanish in one file
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps
1. Open xQuartz and type host (add you IP address)
2. Start docker container for usagov2021
3. In vscode terminal run bin/cypress-ssh
4. In vscode terminal run npx cypress open and the cypress UI should open
5. Goto automated_tests/e2e-cypress/cypress/e2e/regression_testing/nav_card_page.cy.js and verify the test passes in English and Spanish

<img width="1435" alt="Screenshot 2024-09-24 at 10 44 21 AM" src="https://github.com/user-attachments/assets/161a74a0-26e5-4d97-a6a6-58c87a23505f">



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
